content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md

```diff
@@ -37,11 +37,19 @@ Click the timezone dropdown to select a timezone to use for the dashboard. Selec
 
```

Comment:
Should these changes be in both Cloud and OSS docs?

Reply:
@ShmuelLotman is auto-refresh in Both Cloud and OSS?